lib/pluginmanager/update.rb

```diff
@@ -61,7 +65,9 @@ def plugins_to_update(previous_gem_specs_map)
     if update_all?
       previous_gem_specs_map.values.map{|spec| spec.name}
     else
-      not_installed = plugins_arg.select{|plugin| !previous_gem_specs_map.has_key?(plugin.downcase)}
+      # If the plugins isn't available in the gemspec or in 
+      # the gemfile defined with a local path, the plugin is not installed
+      not_installed = plugins_arg.select{|plugin| !previous_gem_specs_map.has_key?(plugin.downcase) && !gemfile.find(plugin) }
```

Comment:
is the call to ```!gemfile.find(plugin)``` sure to find plugins not in the gemfile ? are we suposing some state with this function, so we don't need somehow to check for the ```:path``` option ? 

Related to this, is there of any importance here the ones defined with the :git option?